ci/env/kava-protonet/genesis.json

```diff
@@ -253,12 +253,12 @@
           "istanbul_block": "0",
           "muir_glacier_block": "0",
           "berlin_block": "0",
-          "london_block": "0",
-          "arrow_glacier_block": "0",
-          "gray_glacier_block": "0",
-          "merge_netsplit_block": "0",
-          "shanghai_block": "0",
-          "cancun_block": "0"
+          "london_block": null,
+          "arrow_glacier_block": null,
+          "gray_glacier_block": null,
+          "merge_netsplit_block": null,
+          "shanghai_block": null,
+          "cancun_block": null
         },
         "eip712_allowed_msgs": [],
         "allow_unprotected_txs": false
```

Comment:
See a bunch of other values that should be copied over from https://github.com/Kava-Labs/kava/blob/master/ci/env/kava-internal-testnet/genesis.json

ie [consensus params](https://github.com/Kava-Labs/kava/blob/ee4fc56f013c06a093f3b5a1202652d7e1e81124/ci/env/kava-internal-testnet/genesis.json#L5)

Can do a diff of different sections and defer to the internal testnet values generally